library/src-non-bootstrapped/scala/annotation/internal/experimentalTest.scala

```diff
@@ -0,0 +1,10 @@
+// KEEP IN NON-BOOTSTRAPPED SOURCES
+package scala.annotation
+package internal
+
+/** This is a dummy definition that tests that the stdlib can be compiled with an experimental definition.
+ *  This might be redundant, we keep it definition in case there are no other @experimental definitions in the library.
+ *  As this definition is in `src-non-bootstrapped`, it will not be published.
+ *  It may accidentally be visible while compiling the non-bootstrapped library.
+ */
+@experimental def testExperimental = 4
```

Comment:
Not sure how important it is to test this given that `@experimental` is no longer experimental itself now?

Reply:
This is not really there to check if `@experimental` is experimental, it is to check that we can define experimental definitions in the standard library. This is to make sure we catch a regression as early as possible as this kind of failure could block an update if the reference compiler.

Note that for `3.1.0` we will not have any other experimental definitions. They will all get stabilized.